### ISSUE TITLE
Support macOS native scroll bar shape

### ISSUE THREAD

**Issue by cjwcommuny** [microsoft/vscode#65352]
The native scroll bar of macOS  is:
![image](https://user-images.githubusercontent.com/23581749/50203811-b69cf800-039d-11e9-890a-6daf1045296e.png)
and it can collapse when mouse moves out of the scroll bar.

**Comment by fwcd** [comment]
Both this and rubber banding, as proposed by #14823, would be great to make the editor feel smoother!

**Comment by hd321kbps** [comment]
I propose to redesign the vertical scroll bar.
To reduce width, add volume and shadows.
Liked the option of Sublime Text.
But not only under macOS but under Windows
![sublime-1](https://user-images.githubusercontent.com/9886849/63640264-b8465a80-c6a6-11e9-9e3e-b0282dd61706.jpg)

**Comment by dzpt** [comment]
Any progress?
The current scrollbar is ugly

**Comment by fwcd** [comment]
@dzpt I think this is blocked on an upstream issue: https://bugs.chromium.org/p/chromium/issues/detail?id=546520

**Comment by neekrite** [comment]
I really want that native scroll bar. It's much better looking...

**Comment by dzpt** [comment]
@xeew i've done some tweaks on UI
- rounded scrollbar, popup, inputs, buttons
- separate intellisense icon/func (Atom like)
- fixed some misaligned (padding, spacing) ...
...

they're much more Mac-like with rounded corner.
since VScode UI is windows like with square button, inputs

if you want, i'm going to publish it as an extension.

<img width="883" alt="Screen Shot 2020-06-10 at 13 54 40" src="https://user-images.githubusercontent.com/5781080/84236527-f3285a80-ab21-11ea-8b4f-4c1cddea6bb2.png">
<img width="883" alt="Screen Shot 2020-06-10 at 13 49 40" src="https://user-images.githubusercontent.com/5781080/84236275-962ca480-ab21-11ea-8cc3-87f8196c8601.png">
<img width="883" alt="Screen Shot 2020-06-10 at 13 49 59" src="https://user-images.githubusercontent.com/5781080/84236285-9b89ef00-ab21-11ea-82bf-151b168ddbf2.png">
<img width="883" alt="Screen Shot 2020-06-10 at 13 50 07" src="https://user-images.githubusercontent.com/5781080/84236287-9cbb1c00-ab21-11ea-8aac-1073cdad69ce.png">
<img width="883" alt="Screen Shot 2020-06-10 at 13 53 37" src="https://user-images.githubusercontent.com/5781080/84236416-cd9b5100-ab21-11ea-98ef-e96e39fbea7a.png">

**Comment by xeew** [comment]
@dzpt whoa that's pretty cool! Definitely publish it. And what icon theme are you using?

**Comment by xeew** [comment]
@dzpt when you publish it please tell me about it I really want it

**Comment by dzpt** [comment]
@xeew i'm using Atom icons for above theme.
sure.

**Comment by xeew** [comment]
@dzpt very sorry to annoy you but I've just remembered about that. Are you going to upload it? Thanks.

**Comment by tbrlpld** [comment]
> We closed this issue because we don't plan to address it in the foreseeable future. You can find more detailed information about our decision-making process [here](https://aka.ms/vscode-out-of-scope). If you disagree and feel that this issue is crucial: We are happy to listen and to reconsider.
> 
> If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.
> 
> Thanks for your understanding and happy coding!

I do not understand :/

Why are you overriding the system behaviour with custom scrollbars that have fewer features? The system scollbars are automatically only visible during scrolling, they expand when the mouse is near them making them easier to grab and their background is transparent, which add additional space especially to small windows. 

So I would like to know why you even go through the trouble overriding them?

**Comment by gblazex** [comment]
@dzpt where can we get our hands on this beauty? 😍

**Comment by cloudgazing** [comment]
This is an old issue but I would like to see this added. I've set up my scroll bar the way I want in settings and it functions and looks the same in all apps. I don't see a reason why vscode (and some other apps) needs to have a different one.

There should absolutely be an option in settings for this (if not the default) especially now that win11 has modern and clean looking scroll bars!

**Comment by ozquez** [comment]
@dzpt Did you publish the extension with those UI changes? I would love to use it, it looks amazing!

**Comment by VincentLuders** [comment]
I'd be interested in the extension or the css as well.
